roles/dcos_cli/templates/promdash.json.j2

```diff
@@ -0,0 +1,39 @@
+{
+  "container": {
+    "type": "DOCKER",
+    "docker": {
+      "network": "BRIDGE",
+      "image": "{{ promdash_image }}",
+      "portMappings": [{"containerPort": 3000, "hostPort": 0, "protocol": "tcp"}]
```

Comment:
im pretty sure we also need serviceport here, as we are running it on a bridge, also makes sure that the service all get the same port, when we scale.